src/ui/redux/actions/tags.js

```diff
@@ -0,0 +1,13 @@
+export const doToggleTag = tag => ({
+  type: 'TAG_TOGGLE',
+  data: {
+    tag,
+  },
+});
+
+export const doDeleteTag = tag => ({
```

Comment:
Why delete and toggle? Shouldn't it be just toggle or delete and add?

Reply:
The idea was that sometimes you might want to create a tag, but not follow it. ex: on the publish page

Creating/deleting a tag removes it from the `knownTags`. Deleting a tag also removes it from `followedTags`.

In this flow, you might want to delete recommended tags that you know you won't use again. Or use tags for your content that you never plan to follow.